app_dart/micro_services/results_pipeline/lib/common/logging.dart

```diff
@@ -0,0 +1,7 @@
+// Copyright 2021 The Flutter Authors. All rights reserved.
```

Comment:
Seems like this file should go to common libraries.

Reply:
ditto as above